src/prognostic_equations/implicit/implicit_solver.jl

```diff
@@ -804,8 +804,13 @@ function update_implicit_equation_jacobian!(A, Y, p, dtγ)
                 ᶜdiffusion_h_matrix_scaled,
                 ᶜdiffusion_h_matrix,
             )
-            @. ∂ᶜρq_err_∂ᶜρ =
-                dtγ * ᶜtridiagonal_matrix_scalar ⋅ DiagonalMatrixRow(-(ᶜq) / ᶜρ)
+            if ρq_name == @name(c.ρq_tot)
```

Comment:
```suggestion
            if q_name == @name(q_tot)
```
(Just to be more consistent with other checks)

Reply:
We have two variables `tracer_info` defined, with the first containing `e_tot, q_tot` and the second containing the condensate, precip components; in this instance line 813 within the list of `q_names` in the second tracer_info block shouldn't contain `q_tot`, so I removed this if-else block. See e.g. `789-793` and `803-814` in the latest commit.